<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogudpmulticast</class>
 <widget class="QDialog" name="dialogudpmulticast">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Freeze Tag Menu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(138, 226, 52,255), stop:1 rgba(78, 154, 6,255));</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Z003 [urw]</family>
         <pointsize>84</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(138, 226, 52);</string>
       </property>
       <property name="text">
        <string>F R E E Z E     T A G</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(138, 226, 52);</string>
         </property>
         <property name="text">
          <string>Multicast IPv4 Address:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignHCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_IP">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(138, 226, 52);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="hostOnlySettingsGroupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>58</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(138, 226, 52);</string>
         </property>
         <property name="title">
          <string>Host Only Settings</string>
         </property>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>151</width>
            <height>20</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(138, 226, 52);</string>
          </property>
          <property name="text">
           <string>Seconds of Gameplay:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="secondsGameLineEdit">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>30</y>
            <width>113</width>
            <height>28</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(138, 226, 52);</string>
          </property>
          <property name="text">
           <string>180</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(138, 226, 52);</string>
         </property>
         <property name="text">
          <string>Connected Status:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="connectedTextLabel">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(138, 226, 52);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(138, 226, 52);</string>
         </property>
         <property name="text">
          <string>Pregame Chat Message:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pregameChatLineEdit">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(138, 226, 52);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sendPregameChatButton">
         <property name="styleSheet">
          <string notr="true">background-color:(rgb(138, 226, 52))</string>
         </property>
         <property name="text">
          <string>Send Message to Pregame Chat</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(138, 226, 52);</string>
         </property>
         <property name="text">
          <string>Multicast Port:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignHCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_bind">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(138, 226, 52);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(138, 226, 52);</string>
         </property>
         <property name="text">
          <string>Enter Username:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="usernameEdit">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(138, 226, 52);</string>
         </property>
         <property name="maxLength">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="connectMulticastButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(138, 226, 52);</string>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="disconnectMulticastButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(138, 226, 52);</string>
         </property>
         <property name="text">
          <string>Disconnect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(138, 226, 52);</string>
     </property>
     <property name="title">
      <string>Pregame Chat:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QTextEdit" name="pregameChatTextEdit"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QRadioButton" name="hostRadioButton_2">
          <property name="font">
           <font>
            <pointsize>24</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(138, 226, 52);</string>
          </property>
          <property name="text">
           <string>Host Game</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="joinRadioButton_2">
          <property name="font">
           <font>
            <pointsize>24</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(138, 226, 52);</string>
          </property>
          <property name="text">
           <string>Join Game</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="confirmButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>29</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(138, 226, 52);</string>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <pointsize>6</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Project by: Alexander Cockrell, Jonah Davis, Justina Edwards, Kentrell Martin, Logan Slowik | Fall 2023 | CPE353 with Dr. Coe</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
